src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c

```diff
@@ -6,6 +6,7 @@
 #if defined(TARGET_UNIX)
 #include <dlfcn.h>
 #elif defined(TARGET_WINDOWS)
+#include <icu.h>
```

Comment:
Why did we have to add this despite CoreCLR not needing it?

Reply:
Because if we include #include <windows.h> before, as we use some define that will make windows.h define NTDDI_VERSION < NTDDI_WIN10_RS3, and with that the icu.h will not do anything because in the beginning of this file is checking the NTDDI_VERSION.